DigitalLearningSolutions.Data/Services/PasswordResetService.cs

```diff
@@ -19,6 +19,7 @@ public interface IPasswordResetService
         Task<bool> EmailAndResetPasswordHashAreValidAsync(string emailAddress, string resetHash);
         void GenerateAndSendPasswordResetLink(string emailAddress, string baseUrl);
         Task InvalidateResetPasswordForEmailAsync(string email);
+        public string GenerateResetPasswordHash(User user);

```

Comment:
Do we need the "public" here as everything in interface should be public anyway?